front/src/applications/editor/components/InfraErrors/InfraErrorCorrectorModal.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+
+import { ConfirmModal, useModal } from 'common/BootstrapSNCF/ModalSNCF';
+import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
+import Loader from 'common/Loader';
+import { saveOperations } from 'reducers/editor';
+import { getInfraID } from 'reducers/osrdconf/selectors';
+import { OPERATION_TYPE } from 'applications/editor/tools/types';
+
+const InfraErrorCorrectorModal = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const { closeModal } = useModal();
+
+  const infraId = useSelector(getInfraID);
+  const [isSaving, setIsSaving] = useState(false);
+
+  const { data: infraAutoFixes, isLoading } =
+    osrdEditoastApi.endpoints.getInfraByInfraIdAutoFixes.useQuery(
+      { infraId: infraId as number },
+      { skip: !infraId }
+    );
+
+  function displayFixes() {
+    if (!infraAutoFixes || isLoading || isSaving) {
+      return (
+        <Loader
+          msg={t(
+            `Editor.infra-errors.corrector-modal.${isLoading ? 'loading-loader' : 'saving-loader'}`
+          )}
+        />
+      );
+    }
+    if (infraAutoFixes?.length === 0) {
+      return <div>{t('Editor.infra-errors.corrector-modal.total-fixes_zero')}</div>;
+    }
+    const creationCount = infraAutoFixes.filter(
+      (operation) => operation.operation_type === OPERATION_TYPE.CREATE
+    ).length;
+    const updateCount = infraAutoFixes.filter(
+      (operation) => operation.operation_type === OPERATION_TYPE.UPDATE
+    ).length;
+    const deletionCount = infraAutoFixes.length - creationCount - updateCount;
+    return (
+      <>
+        <div>{t('Editor.infra-errors.corrector-modal.text')}</div>
+        <div>
+          {t('Editor.infra-errors.corrector-modal.create-actions', {
+            creation_count: creationCount,
+          })}
+        </div>
+        <div>
+          {t('Editor.infra-errors.corrector-modal.update-actions', { update_count: updateCount })}
+        </div>
+        <div>
+          {t('Editor.infra-errors.corrector-modal.delete-actions', {
+            deletion_count: deletionCount,
+          })}
+        </div>
```

Comment:
Avoid alone `<div>` it's non sense, use `<p>` perhaps with margin value to do the same display but with explicit semantic.